### PR TITLE
[WPF] Change ContainerStyle to nullable type

### PR DESCRIPTION
**Pull request by almedina-ms**
Changes the ContainerStyle in CollectionElements back to being nullable to avoid the breaking change that may arise from changing it to a not nullable type. Also, I had to create a new converter to solve the issue where it may confuse 'null' value with the first value of an enum

### CODE REVIEW

File: source/dotnet/Library/AdaptiveCards/IgnoreNullEnumConverter.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+                // Try to read regularly
+                return base.ReadJson(reader, objectType, existingValue, serializer);
+            }
+            catch
+            {
+                // Catch invalid values and replace them with default value
+                // Add warning stating behavior
+                Warnings.Add(new AdaptiveWarning((int)WarningStatusCode.UnknownElementType, $"Value \"{reader.Value.ToString()}\" could not be converted to \"{typeof(TEnum).ToString()}\", using null instead."));
+                return null;
+            }
```
Comment (paulcam206) [inline source/dotnet/Library/AdaptiveCards/IgnoreNullEnumConverter.cs]: does this duplicate what we do in the shared model? if a card has `"style": "someRandomString"`, should it be a warning or a parse error?

File: source/dotnet/Test/AdaptiveCards.Test/AllPayloadTests.cs [hunk 2]
```diff
@@ -127,6 +131,9 @@ public void TestAllTestCards()
                     "FallbackParsing",
                     "DeepFallback",
                     "EmptyFallbackCard",
+
+                    // These cards have AdpativeCards with styles on them
+                    "ColumnColumnSetContainer.Bleed.json",
```
Comment (paulcam206) [inline source/dotnet/Test/AdaptiveCards.Test/AllPayloadTests.cs]: >json [](start = 52, length = 4)

is this supposed to have `.json` on the end?

File: source/dotnet/Library/AdaptiveCards/IgnoreNullEnumConverter.cs (hunk 1 above)
Comment (almedina-ms) [reply source/dotnet/Library/AdaptiveCards/IgnoreNullEnumConverter.cs]: I'm following the same pattern as the IgnoreDefaultStringEnumConverter, where a warning is also added, I assumed that's the expected behaviour

---
In reply to: [270215751](https://github.com/Microsoft/AdaptiveCards/pull/2580#discussion_r270215751) [](ancestors = 270215751)

File: source/dotnet/Test/AdaptiveCards.Test/AllPayloadTests.cs (hunk 2 above)
Comment (almedina-ms) [reply source/dotnet/Test/AdaptiveCards.Test/AllPayloadTests.cs]: it shouldn't matter as the comparison made later checks if the filename contains the given string, for consistency I'll delete the ".json"

---
In reply to: [270216378](https://github.com/Microsoft/AdaptiveCards/pull/2580#discussion_r270216378) [](ancestors = 270216378)

**Pull request merged**